client/js/src/client.ts

```diff
@@ -774,7 +776,7 @@ export function api_factory(
 									}
 								}
 							};
-						} else if (protocol == "sse_v1") {
+						} else if (protocol == "sse_v1" || protocol == "sse_v2") {
```

Comment:
Nit: we should add a section in the changeset describing that this release will contain a breaking change for anyone who might be using the SSE protocol directly.

Reply:
Could we also add comments to the code linking to some explanation between `sse_v1` and `sse_v2`, i don't know what these things are.

Reply:
> Nit: we should add a section in the changeset describing that this release will contain a breaking change for anyone who might be using the SSE protocol directly (not through the clients).

Is there any format to documenting a breaking change @abidlabs ?

Reply:
No I would just add a line in the change log